plugin/helpers.go

```diff
@@ -5,6 +5,7 @@ package plugin
 
 import "github.com/mattermost/mattermost-server/model"
 
+// Helpers interface that encompasses methods for plugin helpers to ensure bots and interact with data store
```

Comment:
I think we can generalise this a bit more. It's not just about bots and the store. Helpers in general abstract common patterns away from plugin devs.

Would you also please add:

`Plugins obtain access to the Helpers by embedding MattermostPlugin.`